src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/VariationLicence.kt

```diff
@@ -141,10 +145,17 @@ class VariationLicence(
   standardConditions = standardConditions.toMutableList(),
   additionalConditions = additionalConditions.toMutableList(),
   bespokeConditions = bespokeConditions.toMutableList(),
-  responsibleCom = responsibleCom,
   updatedBy = updatedBy,
 ),
-  Variation {
+  Variation,
+  HasCom {
+
+  @RequiresCom("Do variations always need a COM set? , Does a variation licence always need a COM set?")
+  override fun getCom(): CommunityOffenderManager = responsibleCom
```

Comment:
Yes it does

Reply:
Will leave this in and rejig what is written.